tests/specs/rum/index.e2e.js

```diff
@@ -0,0 +1,37 @@
+import { checkRum } from '../../util/basic-checks'
+import { testAssetRequest } from '../../../tools/testing-server/utils/expect-tests'
+
+describe('basic pve capturing', () => {
+  ['same-origin', 'cross-origin'].forEach(page => {
+    it(`should send rum when ${page} page loads in an iframe`, async () => {
+      const [
+        rumResults
+      ] = await Promise.all([
+        browser.testHandle.expectRum(),
+        browser.url(await browser.testHandle.assetURL(`iframe/${page}.html`))
+      ])
+
+      checkRum(rumResults.request)
+    })
+  })
+
+  it('should capture page load timings', async () => {
+    await browser.testHandle.scheduleReply('assetServer', {
+      test: testAssetRequest,
+      permanent: true,
+      delay: 500
+    })
+
+    const [
+      rumResults
+    ] = await Promise.all([
+      browser.testHandle.expectRum(),
+      browser.url(await browser.testHandle.assetURL('64kb-dom.html'))
+    ])
+
+    checkRum(rumResults.request)
+    expect(parseInt(rumResults.request.query.be, 10)).toBeGreaterThanOrEqual(500)
+    expect(parseInt(rumResults.request.query.fe, 10)).toBeGreaterThanOrEqual(100)
+    expect(parseInt(rumResults.request.query.dc, 10)).toBeGreaterThanOrEqual(100)
```

Comment:
Is 100 specific here or just an arbitrary number?  Would it be more consistent if it just checked that it existed and was greater than `0`?

Reply:
The html asset is delayed by 500ms so it should technically be around 500ms but some browsers are a little troublesome so I just made it 100 to make sure the page was actually being delayed (ie not around 30 40 ms for an instantaneous server response).